src/standalone/Soil/boundary_conditions.jl

```diff
@@ -262,7 +262,7 @@ boundary_var_domain_names(bc::AtmosDrivenFluxBC, ::ClimaLSM.TopBoundary) =
 """
     boundary_var_types(
```

Comment:
do we need to also implement `boundary_var_types` methods for the CoupledAtmosphere case?

Reply:
we will, yeah.. I think we will just need SHF, LHF, and vapor flux. 

the aerodynamic resistance `r_ae` is needed right now because of an issue in SurfaceFluxes.jl (it does not account for surface resistance, so we use both resistances to correct the fluxes surface fluxes returns right now). Longer term I think we can remove this from the cache for all models.

Reply:
(for now, no additional methods for CoupledAtmosphere exist for any model but the bucket model, thought the bucket may rely on defaults and not have special methods like other models may need)